usagov/usagov-2021
# usagov-28-50-state: usagov-28-50-state-scss-lint

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-28

## Description

Adds ability for content managers to create 50-state-pages for the following fields:
field-agriculture-department
field-corrections-dept
field-district-attorneys
field-education-department
field-election-office
field-emergency-management
field-governor
field-health-department
field-local-governments
field-motor-vehicle-office
field-social-services
field-state-attorney-general

Testing Instructions
You’ll need to import config changes
Add taxonomy term ‘State Office Page’ to Page Type
You’ll need to import the state info following amy’s instructions here: https://github.com/usagov/usagov-2021/pull/690
Once you do that the view 50 state pages view should be populated
If you see the states and links in the view preview, create a basic page to test it.
On the basic page
Select ‘State Office Page’ as Page Type
Scroll down to the 50 state pages type and chose one
That should then give you a list of all the states and whatever field you selected from the drop down!
Note if when testing the field looks like the picture below turn off twig debugging. For some reason twig debugging doesn’t allow it to render correctly. But turning it off makes it work!
![225620917-6be8a08d-85c7-4083-a5de-694b95b5a693](https://user-images.githubusercontent.com/5551880/225694964-a71d9a8d-475a-4772-9b13-1cfa2c00bf32.png)

Screen Shot 2023-03-16 at 8 05 34 AM
When correct it should look like the picture below and the go button should take you to an external link.
Screen Shot 2023-03-
![225621477-a5322b03-cbfd-48e2-b94a-bf1617dd0561](https://user-images.githubusercontent.com/5551880/225695013-947b5ac6-2803-4443-b1cc-489443476bb5.png)

16 at 8 46 44 AM

## Checklist for the Developer



## Checklist for the Peer Reviewers
